### DISCUSSION TITLE
wired 模型怎樣自己训练导出onnx？

### DISCUSSION THREAD

**Discussion by nissansz** [RapidAI/TableStructureRec#49] [Q&A]
wired 模型怎樣自己训练导出onnx？

**Comment by Joker1212** [comment]
参考阿里读光哈，wiredv2模型不支持再训练

**Comment by nissansz** [comment]
阿里读光 有训练教程吗？









--------------------------------------------------------------------------------





------------------ 原始邮件 ------------------




 
 发件人: xinyi  ***@***.***>
 

 
 发送时间: 2024-10-14 08:31:24
 

 
 收件人:RapidAI/TableStructureRec ***@***.***> 

 
 抄送:nissanjp ***@***.***>,Author ***@***.***> 

 
 主题: Re: [RapidAI/TableStructureRec] wired 模型怎樣自己训练导出onnx？ (Discussion #49)
 





 参考阿里读光哈，wiredv2模型不支持再训练
